colorama/ansitowin32.py

```diff
@@ -87,14 +87,26 @@ def __init__(self, wrapped, convert=None, strip=None, autoreset=False):
         # to support the ANSI codes.
         conversion_supported = on_windows and winapi_test()
 
+        # if the stream is not closed, check if it's a TTY
+        not_closed = not self.stream.closed
+        isatty = not_closed and self.stream.isatty()
+
+        # check if the FORCE_COLOR environment variable has been set and NO_COLOR has not been set.
+        if 'NO_COLOR' in os.environ:
+            color_allowed = False
+        elif 'FORCE_COLOR' in os.environ:  # overriden by NO_COLOR
+            color_allowed = True
+        else:
+            color_allowed = isatty
```

Comment:
```suggestion
        # check if the FORCE_COLOR environment variable has been set and NO_COLOR has not been set.
        if 'NO_COLOR' in os.environ:
            color_allowed = False
        elif 'FORCE_COLOR' in os.environ:  # overriden by NO_COLOR
            color_allowed = True
        else:
            # can't ask if it's a tty if it's closed
            color_allowed = not self.stream.closed and self.stream.isatty()
```

Reply:
Makes sense. No need to put `isatty` in a variable